charts/refinery/sample-configs/config_complete.yaml

```diff
@@ -84,6 +84,18 @@ LoggingLevel: debug
 UpstreamBufferSize: 10000
 PeerBufferSize: 10000
 
+# QueryAuthToken, if specified, provides a token that must be specified with
+# the header "X-Honeycomb-Refinery-Query" in order for a /query request to succeed.
+# These /query requests are intended for debugging refinery installations and
+# are not typically needed in normal operation.
+# Can be specified in the environment as REFINERY_QUERY_AUTH_TOKEN.
+# If left unspecified, the /query endpoints are inaccessible.
+# QueryAuthToken: "some-random-value"
```

Comment:
It is supposed to be commented out. In this case, if you comment it out, it's more secure and the endpoints don't work. I didn't want to be in a situation of having a default password that people forgot to change. Hopefully, anyone who uncomments it will also change the password.

Thank you for asking!